src/main/java/org/fogbowcloud/manager/core/services/InstantiationInitService.java

```diff
@@ -0,0 +1,97 @@
+package org.fogbowcloud.manager.core.services;
+
+import org.fogbowcloud.manager.core.constants.ConfigurationConstants;
+import org.fogbowcloud.manager.core.plugins.IdentityPlugin;
+import org.fogbowcloud.manager.core.plugins.PluginFactory;
+import org.fogbowcloud.manager.core.plugins.compute.ComputePlugin;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+// TODO: search if it is possible locate *.properties in another path
+@Service("instantiationInitService")
+@PropertySource({
```

Comment:
Is it necessary ? The configuration files are been setted in the method seUpProperties.